### ISSUE TITLE
IP Spoofing/Masking

### ISSUE THREAD

**Issue by simplylate** [arcadiadevs/playerservers-everything#7]
Sending the player's IP allows bad actors to pull their IP, blackmail players, and can lead to harassment. Therefore sending the server a masked IP would be very beneficial. Of course unique to each player to avoid one IP ban wiping out the whole server 😛

Player safety is a giant priority for most networks and this is definitely a must-have feature for a plugin like this.

**Comment by Experimentali** [comment]
Yeah, but make the IP different for each player so IP-Bans still work. Like, use some sort of encoding system to each IP.
